src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Finds a task by its description.
+ */
+public class FindCommand extends Command {
+    private TaskList results;
+    private final String regex;
+
+    /**
+     * Class constructor.
+     *
+     * @param regex Regex to match to.
+     */
+    public FindCommand(String regex) {
+        this.regex = regex;
+    }
```

Comment:
Would suggest putting a line break between the end of this method and the javaDoc for the next to improve readability.